packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -57,6 +69,20 @@ class ScrollPhysics {
   /// is applied recursively and ancestor will appear at the end of the
   /// existing chain of parents.
   ///
+  /// {@tool sample}
+  ///
+  /// In the following example, the [applyTo] method is used to combine two existing [ScrollPhysics] instances,
```

Comment:
The below seems right but I'm not sure what question are we answering for the users. It's a bit academic. What is the concrete practical question the user is likely asking that we're answering here?